src/XMakeCommandLine/UnitTests/CommandLineSwitches_Tests.cs

```diff
@@ -842,7 +842,7 @@ public void SetParameterizedSwitchTests3()
             // parameters, but for testing purposes this is fine
             Assert.True(switches.SetParameterizedSwitch(CommandLineSwitches.ParameterizedSwitch.Logger, "/LOGGER:\"\",asm;\"p,a;r\"", "\"\",asm;\"p,a;r\"", true, false));
 
-            Assert.Equal("/LOGGER:\"\",asm;\"p,a;r\"", switches.GetParameterizedSwitchCommandLineArg(CommandLineSwitches.ParameterizedSwitch.Logger));
+            Assert.Equal("/l:\" /LOGGER:\"\",asm;\"p,a;r\"", switches.GetParameterizedSwitchCommandLineArg(CommandLineSwitches.ParameterizedSwitch.Logger));
```

Comment:
I agree, but that's what's added previously on line [832](https://github.com/Microsoft/msbuild/pull/477/files#diff-c4e03d8af1182dbe57d27455d5f21d78R832)

Probably something to do with the `// don't unquote fake/missing parameters` comment.